### ISSUE TITLE
docxHtml: Wrong spaces inside span

### ISSUE THREAD

**Issue by leouzz** [jsreport/jsreport#1070]
Docx is not properly rendering spaces inside <span>:

```
{{docxHtml content=this}}

{{this}}
```

'this' content:
```
<p>Sintomi <span style="color: rgb(250,197,28);">vari </span><span style="color: 
rgb(163,143,132);">spesso </span><strong>altalenanti prova prova prova </strong>ora meglio?</p>
```

docx:
![Screenshot 2023-06-20 000153](https://github.com/jsreport/jsreport/assets/4449307/8f8bffce-40f8-4fcb-a6fd-cb4accd6e57f)

**Comment by bjrmatos** [comment]
thanks, we are going to check it

**Comment by bjrmatos** [comment]
this was fixed on jsreport `4.0.0`
